libraries/bot-integration-spring/src/main/java/com/microsoft/bot/integration/spring/BotDependencyConfiguration.java

```diff
@@ -127,6 +128,21 @@ public BotFrameworkHttpAdapter getBotFrameworkHttpAdaptor(Configuration configur
         return new BotFrameworkHttpAdapter(configuration);
     }
 
+    /**
+     * Returns the CloudAdapter for the application.
+     *
+     * By default, it uses the {@link CloudAdapter} class.
+     *
+     * @param configuration The Configuration object to read from.
+     * @return A CloudAdapter object.
+     *
+     * @see #getConfiguration()
+     */
+    @Bean
+    public CloudAdapter getCloudAdaptor(Configuration configuration){
+        return new CloudAdapter(configuration);
+    }
```

Comment:
```suggestion
    public CloudAdapter getCloudAdapter(Configuration configuration){
        return new CloudAdapter(configuration);
    }
```

I guess that `Adaptor` is a typo of the other implementation, they can't change it because of... breaking change :(.